article/_posts/2019-07-25-MTAN.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: review
+title:  End-to-End Multi-Task Learning with Attention
+tags:   deep-learning segmentation multi-task-learning attention-maps
+cite:
+    authors: "Shikun Liu, Edward Johns, Andrew J. Davison"
+    title:   "End-to-End Multi-Task Learning with Attention"
+    venue:   "Accepted at Computer Vision and Pattern Recognition (CVPR), 2019"
+pdf:  https://arxiv.org/pdf/1803.10704.pdf
+---
+
+**Code available [here](https://github.com/lorenmt/mtan).**
+
+
+This paper proposes a *Multi-Task Attention Network* (MTAN), an architecture that can be trained end-to-end and can be built upon any feed-foward neural network, simple to implement, and parameter efficient.
+
+![](/article/images/MTAN/proposal.png)
+
+# Introduction 
+
+The authors wanted to perform multiple task simultaneously to avoid training a independent network for each task, not only this approch is more efficient in terms of memory and inference speed, but also seem that related task share informative visual features.
```

Comment:
wanted -> want

Reply:
... not only is this approach ..., but it also seems that related tasks share ...